### DISCUSSION TITLE
how to use?

### DISCUSSION THREAD

**Discussion by zaze06** [mozilla/rhino#1220] [Q&A] [answered]
I'm having issues using the library(?) in it not loading so I cant use it. How would I do it?
I'm using
- IntelliJ 2022.1
- Temurin 17.0.2
- Gradle (unsure of version 7.3 I think)
- rhino 1.7.14

**Comment by p-bakker** [comment]
Unless you provide info about the actual issue(s) you're facing and what avenue you've already explored, I doubt anyone will be able (or willing) to help you

**Reply by zaze06** [reply]
well I whuld if I chuld. Because I don't know what more to add. but I'm sorry for being bad at describing my issue

**Reply by p-bakker** [reply]
If you can't detail what exactly you're trying to achieve and then which exact challenge your running into trying to achieve that goal, i don't think it's possible for anyone to provide any help or guidance

**Reply by zaze06** [reply]
so the issue is that i can't load the engine whit `new ScriptEngineManager().getEngineByName("rhino");`

**Comment by p-bakker** [comment, marked as answer]
If you want to use Rhino through the ScriptEngine interface, you need to make sure you have both the [org.mozilla.rhino-engine](https://github.com/mozilla/rhino/packages/1179320) and 
[org.mozilla.rhino-runtime](https://github.com/mozilla/rhino/packages/1179324) on your classPath and then you should be able to get the engine using the code you already used: `new ScriptEngineManager().getEngineByName("rhino");`

**Reply by zaze06** [reply]
I see. thanks for the help
